core/backend/src/workspace/Workspace.ts

```diff
@@ -70,6 +70,16 @@ export namespace WorkspaceContainer {
     /** attempt to synchronize (i.e. call `checkForChanges`) this cloud container whenever it is connected to a cloud cache. */
     syncOnConnect?: boolean;
   }
+
+  /** A function to supply an [AccessToken]($bentley) for a `WorkspaceContainer`.
+   * @returns a Promise that resolves to the AccessToken for the container.
+   */
+  export type TokenFunc = (
+    /** The properties of the WorkspaceContainer necessary to obtain the access token */
```

Comment:
Will these doc comments on `props` and `accounts` actually show up in the docs?

Reply:
i think so, i'll have to check.

Reply:
reworked the "conventional" way